### PR TITLE
Volunteer Tasks Fixes

### PR DESCRIPTION
**Pull request by constrictingsnake**
# Pull Request

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [X] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

### Code Review Preparation
- [ ] I have performed a self-review and reviewed the changed files
- [ ] My code follows the project's style guidelines and passes linting
- [ ] I have added comments in hard-to-understand areas and updated documentation

### Testing (Ignore for the first PR)
- [ ] I have added relevant tests and they pass locally
- [ ] I have tested the changes manually, including edge cases

### Code Quality & Security
- [ ] I have formatted my code and removed debugging/unused code
- [ ] No TypeScript errors or linting issues
- [ ] No sensitive information committed (API keys, passwords, etc.)
- [ ] User inputs are validated and sanitized

### Dependencies & Deployment
- [ ] Dependencies are documented and package-lock.json is updated if needed
- [ ] Changes work with Firebase emulator and environment variables are documentedRetry

## Screenshots (If it is a front end feature screenshot is required)

image

image

## Summary by CodeRabbit

* **New Features**
  - Volunteer tasks now open to the shift list by default
  - New empty-state illustration when no eligible shifts are available

* **Improvements**
  - Responsive toast placement for better visibility on different screen sizes
  - Inventory flows: per-item removal, selection sets explicit quantity (no auto-increment), and updated action labels ("Modify"/"Add to Inventory")
  - Search/list UI: improved scrolling, numeric-only quantity input, icon scaling and minor label/capitalization tweaks

### CODE REVIEW

File: app/volunteer-tasks/page.tsx [hunk 1]
```diff
@@ -44,12 +45,15 @@ export default function VolunteerTasks() {
 
     const now = Date.now();
     const userTimeBlocks = allTB
-        .filter((tb) => tb.volunteerGroups.some((g) => g.volunterIDs.includes(currentUser!.uid)))
+        .filter((tb) =>
+            tb.endTime.toMillis() >= now &&
+            tb.volunteerGroups.some((g) => g.volunterIDs.includes(currentUser!.uid))
```
[the remark this answers is not in the document]
Comment (constrictingsnake) [reply app/volunteer-tasks/page.tsx:52]: user alr gonna be authed at this point

**Review by joelchem — APPROVED** [review]
